lib/jsdom/living/xhr-sync-worker.js

```diff
@@ -4,6 +4,10 @@ const util = require("util");
 
 const jsdom = require("../../jsdom");
 const xhrSymbols = require("./xmlhttprequest-symbols");
+const tough = require("tough-cookie");
+
+const doc = jsdom.jsdom();
```

Comment:
To define these sooner but it does not change anything, I can put these back to the "end" callback if you prefer.